code/modules/power/breaker_box.dm

```diff
@@ -38,7 +38,7 @@
 
 	busy = 1
 	to_chat(user, "<span class='green'> Updating power settings..</span>")
-	if(do_after(user, 50, FALSE, 5, BUSY_ICON_GENERIC)) //5s for AI as AIs can manipulate electronics much faster.
```

Comment:
Is the removal of the icon intended here?

Reply:
It's done remotely by an AI, yea. this would make no sense for those around said AI.